Dockerfile.assets-builder

```diff
@@ -61,4 +56,4 @@ RUN gulp --environment ${environment} js
 COPY --chown=elife:elife source/ ${PROJECT_FOLDER}/source
 
 COPY --chown=elife:elife assets/sass ${PROJECT_FOLDER}/assets/sass
-RUN gulp --environment ${environment} generateStyles
+RUN gulp --environment ${environment} generateCss
```

Comment:
this is the only external reference to this gulp task

Reply:
We shouldn't globally install Gulp, and instead use the local version.

Reply:
Yes, can follow up with a tidy up.

Reply:
I've renamed that gulp task to `generateCss` it as it's doing a slightly different job now. It runs as part of the overall gulp watch.